model/manifest.go

```diff
@@ -168,6 +169,10 @@ type ManifestServer struct {
 	// If your plugin is compiled for multiple platforms, consider bundling them together
 	// and using the Executables field instead.
 	Executable string `json:"executable" yaml:"executable"`
+
+	// Static files to be served at http://$SITE_URL/plugin/$PLUGIN_ID/public/{anything}
+	// This should be relative to the root of your bundle and the location of public files
+	StaticFiles string `json:"static_files" yaml:"static_files"`
```

Comment:
You are correct, I was keying off of that conversation. I figured I'd add it, and if we didn't need it, then it is easily removed. It would simplify the code.

Reply:
Re-pinging @lieut-data...

Reply:
My worry was accidentally exposing a folder named `public` in someone's plugin directory that wasn't meant to be /this/ public (maybe it had some other purpose). Making it part of the manifest would require an explicit opt-in as such.

That being said, I suspect this is over-thinking the problem, and we should start with the simpler variant as you propose.

Reply:
Coming back to this, I think I disagree with myself: we provide configuration for the `bundle_path` and `executable`, even though they are the same in all cases. I think it makes sense to do the same thing here, and preserve the functionality implemented thus far.

Reply:
👍

Reply:
I thought the reason `bundle_path` and `executable` are configurable is, that it's easy to determinate if a plugin has a server or webapp part. Looking back this there might be better ways to do this.

To me plugins are in an phase where consistency isn't a huge priority. I would like to keep things a unified as possible. At the end this feature is just an offer to plugin authors. They can implement there own method of doing this if they don't want to use this feature.

Reply:
👍